mitodl/micromasters
# Financial Aid: Implemented endpoint for retrieving a learner's course price

#### What are the relevant tickets?
Fixes #1044 

#### What's this PR do?
Implements the API endpoint for retrieving a learner's course price as well as information about their potential financial aid request

#### Where should the reviewer start?
This PR depends on changes made in #1096, so most of the commits/files changed are duplicated in that PR.  I've left this without a "Needs Review" label for now until the other PR is merged.

#### How should this be manually tested?
You can visit api/v0/financial_aid_course_price/<user_id>/<program_id> to use DRFs built-in helper pages to check input and output.

#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### What GIF best describes this PR or how it makes you feel?
